Automattic/Gravatar-SDK-Android
# Show empty state when no avatars are uploaded

Closes #252

### Description
This PR implements the empty state screen for when there are no avatars uploaded. 

| Light | Dark |
|---|---|
| <img width="677" alt="Screenshot 2024-08-16 at 15 35 31" src="https://github.com/user-attachments/assets/8cb1a380-05d7-46f0-9505-e03a0a4a8245"> | <img width="677" alt="Screenshot 2024-08-16 at 15 35 23" src="https://github.com/user-attachments/assets/228d9eb5-7b12-4296-9f92-d4415cf6b20e"> |

### Testing Steps
**Important Note**: The backend is currently returning the wrong format for the `updated date` field. As a workaround, you can apply this patch to comment that field from the Avatar object.
[Gravatar-Android-13-04-48.patch](https://github.com/user-attachments/files/16455284/Gravatar-Android-13-04-48.patch)

1. Make sure you have zero avatars uploaded
2. Run the DemoApp 
3. Go to Avatar Update tab
4. Tap `Update Avatar` button
5. Follow the steps to log in (if necessary)
6. Confirm you see the empty state
7. Upload an image
8. Confirm you see the list
